### PR TITLE
refactor(avatar): flag for size

### PR DESCRIPTION
**Pull request by Stitch07**

### CODE REVIEW

File: src/commands/Tools/avatar.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 import { SkyraCommand } from '@lib/structures/SkyraCommand';
-import { MessageEmbed } from 'discord.js';
+import { MessageEmbed, ImageExt, ImageSize } from 'discord.js';
 import { CommandStore, KlasaMessage, KlasaUser } from 'klasa';
 
+const VALID_SIZES = [32, 64, 128, 256, 512, 1024, 2048];
+
 export default class extends SkyraCommand {
 
+	private kExtensionRegex = /.?(png|jpg|webp|gif)/i;
```
Comment (vladfrangu) [inline src/commands/Tools/avatar.ts]: Escape the dot and allow jpeg
```suggestion
	private kExtensionRegex = /\.?(png|jpe?g|webp|gif)/i;
```

File: src/commands/Tools/avatar.ts [hunk 2]
```diff
@@ -13,20 +17,37 @@ export default class extends SkyraCommand {
...
+			.setAuthor(user.tag, user.displayAvatarURL({ size: 128, format: ext, dynamic: true }))
 			.setColor(await DbSet.fetchColor(message))
-			.setImage(user.displayAvatarURL({ size: 2048, format: 'png', dynamic: true })));
+			.setImage(user.displayAvatarURL({ size, format: ext, dynamic: true })));
+	}
+
+	private resolveExt(ext: string): ImageExt {
+		const results = this.kExtensionRegex.exec(ext);
+		this.kExtensionRegex.lastIndex = 0;
+		if (results) return results[0].toLowerCase() as ImageExt;
```
Comment (vladfrangu) [inline src/commands/Tools/avatar.ts]: For a string like `png`, this is fine, but for one like `.png`, it isn't 
image

```suggestion
		if (results) return results[1].toLowerCase() as ImageExt;
```

**Review by vladfrangu — DISMISSED** [review]

File: src/commands/Tools/avatar.ts (hunk 2 above)
Comment (Stitch07) [reply src/commands/Tools/avatar.ts]: I've talked to Favna and I'm removing the extension flag entirely, so this will not matter.

**Review by kyranet — DISMISSED** [review]
`--ext` was removed but is still referenced in the i18n strings.

**Review by favna — APPROVED** [review]

**Pull request merged**